bot/handlers.py

```diff
@@ -0,0 +1,541 @@
+from typing import NoReturn
+
+from telegram import ParseMode
+from telegram import Update
+from telegram.ext import (
+    CallbackContext,
+    CommandHandler,
+    Filters,
+    MessageHandler,
+    CallbackQueryHandler,
+    ConversationHandler,
+)
+
+import buttons
+import constants
```

Comment:
Absolute import required. We need to read [pep8 import instructions](https://www.python.org/dev/peps/pep-0008/#imports) together